### PR TITLE
Support widgets=print parameter to init app with print panel open

### PR DESCRIPTION
**Pull request by gjn**
Fix for https://github.com/geoadmin/mf-geoadmin3/issues/2141

### CODE REVIEW

File: src/js/MainController.js [hunk 1]
```diff
@@ -114,7 +116,10 @@
             isWindowTooSmall()) {
           showCatalog = false;
         }
-        $scope.globals.catalogShown = showCatalog;
+        if (!initWithPrint) {
+          $scope.globals.catalogShown = showCatalog;
+          initWithPrint = false;
```
Comment (oterral) [inline src/js/MainController.js]: initWithPrint is already false here

Comment (oterral) [inline src/js/MainController.js]: On the next topi change the print widget stays open and the catalog is always closed. I don't know if it's wanted or not

Comment (gjn) [inline src/js/MainController.js]: The if is wrong...I'll adapt.

**Comment by gjn** [comment]
@oterral It's corrected. Catalog change now opens catalog (and closes print).

**Comment by oterral** [comment]
Perfect thx

**Pull request merged**